components/admin_console/__snapshots__/admin_console.test.jsx.snap

```diff
@@ -214,6 +214,6 @@ exports[`components/AdminConsole should generate the routes 1`] = `
 
 exports[`components/AdminConsole should redirect to / when not system admin 1`] = `
```

Comment:
I guess redirecting to `/` is not the case anymore.

Reply:
👍